### ISSUE TITLE
D403 not being automatically fixed

### ISSUE THREAD

**Issue by alexdauenhauer** [charliermarsh/ruff#4759]
According to the [pydocstyle section](https://beta.ruff.rs/docs/rules/#pydocstyle-d) code D403 is fixable, but it does not get fixed for me using the vs code extension or command line

**code**
```python
    @abstractproperty
    def features(self) -> Dict[str, Any]:
        """get features required for model inference and wrap in a dictionary"""
```

**expected**
I expected the first word in the docstring to be capitalized.

When I run flake8 it flags D403 on this line. when I run ruff either with the VS Code extension or on the command line via `ruff check` it does not flag this line and it does not auto fix it

**ruff version**
vs code extension v2023.16.0

**ruff settings**
```json
"ruff.args": [
        "--line-length=100",
],
```

**Comment by charliermarsh** [comment]
The `D` rules aren't enabled by default, is it possible that the `D` rules haven't been enabled at all for your project?

For example, if I run with `--isolated`, I can get Ruff to fix that violation:

```shell
❯ ruff check foo.py --isolated --select D403 --fix
Found 1 error (1 fixed, 0 remaining).
```

**Comment by alexdauenhauer** [comment]
@charliermarsh oh I see I was misunderstanding the flags. `select` is what I should have been looking at instead of `fixable`. I was able to achieve the autofix with vs code by setting `--extend-select=D403`. If that looks right to you then I'll close this issue

**Comment by charliermarsh** [comment]
Ahhh yes, sorry about that! It does sound right.
